components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/LoginEventHookHandler.java

```diff
@@ -54,13 +54,19 @@
  */
 public class LoginEventHookHandler extends AbstractEventHandler {
 
-    private static final Log log = LogFactory.getLog(LoginEventHookHandler.class);
+    private final Log log;
```

Comment:
why do we move with the instance variable. IMO it should not be some variable accepted at the constructor